src/pages/workspace/tags/WorkspaceTagsSettingsPage.tsx

```diff
@@ -66,6 +67,7 @@ function WorkspaceTagsSettingsPage({route, policyTags}: WorkspaceTagsSettingsPag
                                         isOn={policy?.requiresTag ?? false}
                                         accessibilityLabel={translate('workspace.tags.requiresTag')}
                                         onToggle={updateWorkspaceRequiresTag}
+                                        disabled={!policy?.areTagsEnabled || !hasEnabledOptions}
```

Comment:
If I enable "Members must tag all expenses" then disable all the tags, then the "Members must tag all expenses" switch is locked at ON.


https://github.com/Expensify/App/assets/16493223/db76ba22-2938-41de-958e-cc24fa642770


Solution: If tagging is required, the user shouldn't be able to disable the last enabled tag

Reply:
Could be done in a follow up

Reply:
Maybe even here? https://github.com/Expensify/App/pull/42328

Reply:
Yep I think https://github.com/Expensify/App/pull/42328 is going to solve the issue. let me double check on that